signbank/dictionary/tests/test_csv_import.py

```diff
@@ -564,3 +565,222 @@ def test_confirmation_view_no_post_method(self):
         response = self.client.get(reverse('dictionary:confirm_import_qualtrics_csv'))
         self.assertEqual(response.status_code, 302)
         self.assertRedirects(response, reverse("dictionary:import_qualtrics_csv"))
+
+
+class ManualValidationCSVImportTestCase(TestCase):
+    def setUp(self):
+        # Create user and add permissions
+        self.user = User.objects.create_user(username="test", email=None, password="test")
+        csv_permission = Permission.objects.get(codename='import_csv')
+        self.user.user_permissions.add(csv_permission)
+
+        # Create client with change_gloss permission.
+        self.client = Client()
+        self.client.force_login(self.user)
+
+        # Create user with no permissions
+        self.user_noperm = User.objects.create_user(username="noperm", email=None,
+                                                    password="noperm")
+        self.client_noperm = Client()
+        self.client_noperm.force_login(self.user_noperm)
+
+        # Create client not logged in
+        self.client_nologin = Client()
+
+        # Create a gloss
+        # Migrations have id=1 already
+        self.signlanguage = SignLanguage.objects.create(pk=2, name="testsignlanguage",
+                                                        language_code_3char="tst")
+        self.dataset = Dataset.objects.create(name="testdataset", signlanguage=self.signlanguage)
+        self.gloss_1 = Gloss.objects.create(idgloss="testgloss:1", dataset=self.dataset)
+        self.gloss_2 = Gloss.objects.create(idgloss="testgloss:2", dataset=self.dataset)
+
+        # Assign view permissions to dataset for user
+        assign_perm('view_dataset', self.user, self.dataset)
+
+    # Unimportant columns are excluded from csv
```

Comment:
**question**: what does this comment mean?

Reply:
It means I could add more columns to the test file, but they will be ignored during the import. Reason for this was that there is a possibility of the ready-for-validation csv being used and extended for the manual validations. In which case I'd ignore the video_url and gloss_main translations column

Reply:
Sorry, I should have phrased that more clearly 😅

I meant that the comment didn't seem clear without having read the implementation. I was wondering if it might be worth adding the context that the csv _**can**_ contain other headers, but that any other columns will be ignored